pkg/virt-launcher/virtwrap/device/hostdevice/sriov/hostdev_test.go

```diff
@@ -268,6 +373,14 @@ func newSRIOVAlias(netName string) *api.Alias {
 	return api.NewUserDefinedAlias(netsriov.AliasPrefix + netName)
 }
 
+func newSRIOVInterfaceWithBootOrder(name string, bootOrder int) v1.Interface {
```

Comment:
I would generate the name form the boot order, it will reduce some noise form the caller